tests/test_versioning.py

```diff
@@ -22,11 +54,31 @@ class Xv2Serializer(serializers.Serializer):
     id = serializers.UUIDField()
 
 
+class Person(models.Model):
```

Comment:
unfortunately you cannot reuse model names in the tests and Person is already taken. use something else

Reply:
Done. Sorry for the unprepared code.